install/empty_data.php

```diff
@@ -940,6 +941,37 @@ public function getEmptyData(): array
             $i++;
         }
 
+        // Placeholder dashboards
+        $tables['glpi_dashboards_dashboards'][] = [
+            'id' => $i,
+            'is_placeholder' => 1,
+            'key' => 'central_placeholder',
+            'name' => 'Example central',
```

Comment:
I think it should be translated.
```suggestion
            'name' => __('Example central'),
```